src/configure.ac

```diff
@@ -233,6 +233,8 @@ fi
    JTR_FLAG_CHECK([-Wall], 1)
    JTR_FLAG_CHECK([-Wdeclaration-after-statement], 1)
 
+   JTR_FLAG_CHECK([[-Wno-stringop-truncation]], 1)
```

Comment:
This patch does not work as intended because:
```
When an unrecognized warning option is requested (e.g., ‘-Wunknown-warning’),
GCC emits a diagnostic stating that the option is not recognized. However, if the
‘-Wno-’ form is used, the behavior is slightly different: no diagnostic is produced for
‘-Wno-unknown-warning’ unless other diagnostics are being produced.
```

This option requires gcc 8. So, gcc 4 [1] is lying to you:
```
checking if gcc supports -Wno-stringop-truncation... yes
```

[1] gcc (GCC) 4.4.7 20120313 (Red Hat 4.4.7-23)

Reply:
OK, so we have to test for `-Wstringop-truncation` and if that is supported, we need to add `-Wno-stringop-truncation`. I recall we do something like that for some other warning?

Perhaps we should write a special JTR_FLAG_CHECK function for this use case.

Reply:
I'm working on this!

Reply:
I have a feeling that could be slightly less portable.